AODN/NSW-OEH/incoming_handler.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+export PYTHONPATH="$DATA_SERVICES_DIR/lib/python"
+export SCRIPTPATH="$DATA_SERVICES_DIR/AODN/NSW-OEH"
+
+declare -r BACKUP_RECIPIENT=marty.hidas@utas.edu.au
+
+
+# notify_uploader - send email report to uploader of $INCOMING_FILE and log where it was sent
+# send a copy of the email to $BACKUP_RECIPIENT as well
+# $1 - file containing the message body
+# $2 - message subject
+notify_uploader() {
+    local report=$1; shift
+    local subject="$1"; shift
+
+    local uploader_email
+    uploader_email=`get_uploader_email $INCOMING_FILE`
+    if [ -n "$uploader_email" ]; then
+        cat $report | notify_by_email $uploader_email "$subject"
+        log_info "Email report sent to '$uploader_email'"
+    fi
+
+    cat $report | notify_by_email $BACKUP_RECIPIENT "$subject"
+    log_info "Email report sent to '$BACKUP_RECIPIENT'"
+}
+
+# main - handle a zip file containing data and metadata for
+# one survey from NSW-OEH
+# $1 - zip file to handle
+main() {
+    local zipfile=$1; shift
+
+    # set up temp unzip directory and report file
+    local unzip_dir=`mktemp -d --tmpdir ${JOB_NAME}_XXXXX`
+    chmod 755 $unzip_dir  # make directory world readable
```

Comment:
I think this was mainly for debugging purposes. Back when we were first developing the pipelines we often ended up with these temp directories left there after a pipeline process ended and we couldn't even see what was in them.